packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -1818,10 +1818,19 @@ class _JsonViewerState extends State<JsonViewer>
     with ProvidedControllerMixin<DebuggerController, JsonViewer> {
   late Future<void> _initializeTree;
   late DartObjectNode variable;
+  Future<void> _buildAndExpand(
+    DartObjectNode variable,
+  ) async {
+    // Build the root node
+    await buildVariablesTree(variable);
+    // Build the contents of all children
```

Comment:
Does this only build the second level of children (i.e., not recursively down the tree)?

Reply:
correct